crates/server/src/sse/storage.rs

```diff
@@ -0,0 +1,58 @@
+use calimero_server_primitives::sse::ConnectionId;
+use calimero_store::key::Generic as GenericKey;
+use calimero_store::layer::{ReadLayer, WriteLayer};
+use calimero_store::slice::Slice;
+use calimero_store::Store;
+use eyre::Result as EyreResult;
+
+use super::config::SSE_SESSION_SCOPE;
+use super::session::PersistedSessionData;
+
+/// Generate storage key for a session
```

Comment:
I think it's important to add that info here, so it's clear the intention and the content of the function.
I'd also suggest to rename it to `session_storage_key` that reflects its purpose better.

```suggestion
/// Generates a generic storage key for a given session ID.
///
/// This function creates a unique key by taking the `session_id`, converting it
/// to its big-endian byte representation, and using it as the first 8 bytes
/// of the key's fragment. The remaining bytes of the fragment are zero-padded.
/// The entire key is scoped under `SSE_SESSION_SCOPE` to prevent collisions.
///
/// # Arguments
///
/// * `session_id` - The `ConnectionId` for which to generate a storage key.
///
/// # Returns
///
/// A `GenericKey` that can be used to uniquely identify session data in a storage backend.
```